mozilla_django_oidc/views.py

```diff
@@ -114,15 +114,15 @@ def redirect_url(self):
 
     def dispatch(self, request, *args, **kwargs):
         """Log out the user."""
-
-        if request.user.is_authenticated():
+        # Check if a method exists to build the URL to log out the user
+        # from the OP.
+        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
```

Comment:
Done. I don't think it's glorious but it's a start and at least things will not raise UnboundLocalError if you sign out as anonymous.